roq-frontmatter/deployment/src/main/java/io/quarkiverse/roq/frontmatter/deployment/TemplateLink.java

```diff
@@ -81,7 +82,7 @@ public static String resolveSlug(LinkData data) {
                 title = baseFileName;
             }
         }
-        return slugify(title).toLowerCase();
```

Comment:
I'm a _little_ concerned by this change, as the behavior of the method is changed, but it doesn't look like anything else in Roq itself is calling this, so it may be fine. Breaking this apart to preserve the behavior while implementing the feature seemed a little overkill, but I'm happy to do that if you feel we need to preserve behavior here.